### PR TITLE
Updated missing documentation for cluster remap command

### PR DESCRIPTION
**Pull request by prajin-29**
The `cluster-remap` and `revert-cluster-remap` commands have been added to the project's open-source library to assist administrators in managing their clusters' UC settings. The `cluster-remap` command allows users to remap their clusters to UC-enabled ones by providing a comma-separated list of cluster IDs. It updates the specified clusters to UC-enabled and stores a backup of the existing cluster configuration in a JSON file. This backup can be used to revert the cluster remapping if necessary using the `revert-cluster-remap` command. This command iterates through the list of clusters from the backup folder and reverts the cluster configurations to their original state. These commands provide administrators with the ability to manage their clusters' UC settings and revert any changes made to the cluster configurations if necessary, ensuring the integrity and flexibility of their systems.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -715,6 +717,43 @@ This command is useful for administrators who want to ensure that the groups hav
...
+```text
+Please provide the cluster id's as comma separated value from the above list (default: <ALL>): 
+```
+
+This command will remap the cluster to uc enabled one.When we run this command it will list all the clusters 
+and its id's and asks to provide the cluster id's as comma separated value which has to be remapped, by default it will take all cluster ids.
+Once we provide the cluster id's it will update these clusters to UC enabled.Back up of the existing cluster 
+config will be stored in backup folder inside the installed location(backup/clusters/cluster_id.json) as a json file.This will help 
+to revert the cluster remapping.
+
```
Comment (nfx) [inline README.md:738]: missing `back to top` links.

File: README.md [hunk 2]
```diff
@@ -53,6 +53,8 @@ See [contributing instructions](CONTRIBUTING.md) to help improve this project.
   * [`manual-workspace-info` command](#manual-workspace-info-command)
   * [`create-account-groups` command](#create-account-groups-command)
   * [`validate-groups-membership` command](#validate-groups-membership-command)
+  * [`cluster-remap` command](#cluster-remap)
+  * [`revert-cluster-remap` command](#revert-cluster-remap)
```
Comment (nfx) [inline README.md:57]: this anchor is invalid and non-clickable. regenerate with markdown IDE plugin:
image

**Review by nfx — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (prajin-29) [reply README.md]: Updated the command

**Review by nfx — APPROVED** [review]
lgtm

**Pull request merged**